custom_components/alexa_media/media_player.py

```diff
@@ -1448,17 +1453,18 @@ async def async_play_media(self, media_type, media_id, enqueue=None, **kwargs):
             "options"
         ].get(CONF_PUBLIC_URL, DEFAULT_PUBLIC_URL)
         if media_type == "music":
-            if public_url:
-                await self.async_play_tts_cloud_say(public_url, media_id, **kwargs)
-            else:
-                await self.async_send_tts(
-                    "To send TTS, set public url in integration configuration"
-                )
+            if not public_url:
+                # Log and notify for missing public URL
                 _LOGGER.warning(
-                    "To send TTS, set public url in integration configuration"
-                    " Please see the alexa_media wiki for details."
-                    "https://github.com/alandtse/alexa_media_player/wiki/Configuration%3A-Notification-Component#use-the-notifyalexa_media-service"
+                    "To send TTS, please set the public URL in integration configuration."
```

Comment:
Same thing about duplicate string please. I recognize you are just correcting what was already there.